dm/ui/.husky/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+export PATH="$PATH:$(pwd)/dm/ui/node_modules/.bin"
+linter=$(pwd)/dm/ui/node_modules/.bin/lint-staged
+if test -f "$linter"; then
```

Comment:
lint-check should move into CI in case other develop not install pre-commit plugin locally

Reply:
I'll add a linter job for github action later, not intend to do too much in one single pull request

Reply:
no need to add to github action, just add it to `make-check` step in Makefile

Reply:
@Ehco1996 is node and yarn preinstalled in CI images?

Reply:
currently no,  but we can build/download linter bin to `bin/` dir like [golangci-lint](https://github.com/pingcap/ticdc/blob/1a285a478ddf058ede2d20c59f0850c0b6a1e35a/Makefile#L372)